src/pat/modal/modal.js

```diff
@@ -58,7 +58,7 @@ export default Base.extend({
                 '    <div class="<%= options.classModal %>" role="document">' +
                 '      <div class="<%= options.classHeaderName %>">' +
                 '        <% if (title) { %><h5 class="modal-title" id="modal-title" tabindex="0"><%= title %></h5><% } %>' +
-                '        <button type="button" class="btn-close modal-close" aria-label="Close"></button>' +
+                '        <button type="button" class="btn-close modal-close" aria-label="<%= _t("Close") %>"></button>' +
```

Comment:
Unfortunately this doesn't work. You need to define a property, for example `tplObject.closeLabel` with the translated label and use it in the template like `<%= closeLabel %>`. See https://github.com/plone/mockup/blob/23f4d61cff105e946fea7feb5202da12bdce6ffe/src/pat/modal/modal.js#L392-L399 how its done with the other vars.